src/main/java/net/snowflake/client/jdbc/DefaultSFConnectionHandler.java

```diff
@@ -158,18 +155,11 @@ private void initSessionProperties(SnowflakeConnectString conStr) throws SFExcep
     }
   }
 
-  /**
-   * Get an instance of a ResultSet object
-   *
-   * @param queryID
-   * @return
-   * @throws SQLException
-   */
   @Override
-  public ResultSet createResultSet(String queryID, Connection connection) throws SQLException {
```

Comment:
To reviewers: this is a 'vestigial' change from some feedback provided in an earlier PR. We change this to take `Statement`, as it is cleaner than passing `Connection`.